proof/refine/Invariants_H.thy

```diff
@@ -1800,26 +1483,6 @@ lemma obj_at'_pspaceI:
   "obj_at' t ref s \<Longrightarrow> ksPSpace s = ksPSpace s' \<Longrightarrow>  obj_at' t ref s'"
   by (auto intro!: projectKO_stateI simp: obj_at'_def ps_clear_def)
 
-lemma cte_wp_at'_pspaceI:
-  "\<lbrakk>cte_wp_at' P p s; ksPSpace s = ksPSpace s'\<rbrakk> \<Longrightarrow> cte_wp_at' P p s'"
-  apply (clarsimp simp add: cte_wp_at'_def getObject_def)
```

Comment:
I would have expected this one also to be a generic lemma. Does it need any arch specific ingredients?

Reply:
I was able to prove this with better tcb_cte_cases infrastructure. next round

Reply:
done